DEVELOPING.md

```diff
@@ -83,61 +83,9 @@ if *debugging {
 }
 ```
 
-## Git Workflow
-1.  Fork in the cloud
-    1.  Visit https://github.com/google/go-flow-levee
-    2.  Click `Fork` button (top right) to establish a cloud-based fork.
+## Git workflow
 
-2.  Clone fork to local storage
-
-    ```bash
-    cd $(go env GOPATH)/src/
-    git clone https://github.com/<your-github-username>/go-flow-levee.git
-
-    cd go-flow-levee
-    git remote add upstream https://github.com/google/go-flow-levee.git
-
-    # Never push to upstream master
-    git remote set-url --push upstream no_push
-
-    # Confirm that your remotes make sense:
-    git remote -v
-    ```
-
-3.  Branch
-
-    Get your local master upto date:
-
-    ```bash
-    cd $(go env GOPATH)/src/go-flow-levee
-    git fetch upstream
-    git checkout master
-    git merge upstream/master
-    ```
-
-    Branch from it:
-    ```bash
-    git checkout -b featurebranch
-    ```
-
-4.  Keep your branch in sync
-    ```bash
-    git fetch upstream
-    git merge upstream/master
-    ```
-
-5.  Pushing your commits
-    ```bash
-    git push origin featurebranch
-    ```
-
-    **Note:** Please avoid force-pushing, as it can break links to commits and cause GitHub to lose track of comment threads.
-
-6.  Create a Pull Request
-    1.  Goto https://github.com/google/go-flow-levee
-    2.  Click on the "Compare & pull request" button.
-
-    **Note:** When merging your PR, please use a squash merge.
+To see the preferred git workflow see the [git workflow guide](GIT_WORKFLOW.md)
```

Comment:
This works for me. I think the `**Note:**` comments should be repeated here though.

Reply:
I prefer to have this information all in one location. Adding the notes here without all the other context and commands doesn't seem right.